client/css/style.css

```diff
@@ -666,6 +666,11 @@ button, .user {
 #chat.hide-quit .quit {
 	display: none !important;
 }
+#chat .invite .from:before {
+	font-family: FontAwesome;
+	content: "\f0e0";
```

Comment:
FYI, this is the envelope icon.
Throwing in out there, what about the `user-plus` icon instead: https://fortawesome.github.io/Font-Awesome/icon/user-plus/ ? Any opinions?